parking-lot-help-request/firmware/README.md

```diff
@@ -0,0 +1,99 @@
+# Parking Lot Help Request Firmware
+
+This project is meant to be host-free because the Notecard itself can be configured to send notes to Notehub when the help button is pressed.
+
+Instead of a true firmware set up guide, the [first part](#configure-the-notecard) of this document will guide you through creating a reusable Notecard configuration script which can be executed through the [Notecard CLI](https://dev.blues.io/tools-and-sdks/notecard-cli/) or through the [online webREPL](https://dev.blues.io/notecard-playground/) available on the Blues Developer Experience site.
```

Comment:
Not that it matters much (or at all), but I think set up → setup is correct here.

Reply:
It does matter. Small grammatical mistakes like that drive me nuts when I see them - keep pointing them out.

Your the best ;)